src/config.js

```diff
@@ -1,45 +1,60 @@
 import { base } from '@tokend/js-sdk'
 
-const featureFlags = {
-  dashboard: true,
-  fees: true,
-  operations: true,
-  issuance: true,
-  assets: true,
-  requests: true,
-  settings: true,
-}
-
 export default Object.assign(
   {
-    DEBUG: true,
-    HORIZON_SERVER: process.env.HORIZON_SERVER,
+    /**
+     * URL of the Horizon server. The application is going to fetch all the
+     * needed network configs from the provided value.
+     */
+    HORIZON_SERVER: '',
+
+    /**
+     * URL of the storage server. Here will be stored pics and other uploadable
+     * files.
+     */
     FILE_STORAGE: '',
-    NETWORK_PASSPHRASE: '',
-    TRANSACTIONS_PER_PAGE: 12,
-    REQUESTS_PER_PAGE: 10,
-    DECIMAL_POINTS: 6,
-    MINIMAL_NUMBER_INPUT_STEP: 0.000001,
-    VALIDATE_EMAILS: process.env.NODE_ENV === 'production',
-    FEATURE_FLAGS: featureFlags,
-    NULL_ASSET_SIGNER: 'GAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHV4',
-    LOCAL_STORAGE_KEY: 'tokend-client',
-    DEFAULT_QUOTE_ASSET: 'USD',
-    DEFAULT_TRADE_PAIRS_RE: [
-      // Descending priority
-      /BTC.*\/.*ETH/,
-      /ETH.*\/.*BTC/,
-    ],
+
+    // TODO: remove
+    DEBUG: true,
+
+    /**
+     * Will skip the email confirmation step during sign up. Instead of the
+     * sipped step, the user will be instantly logged in and redirected
+     * to the application.
+     */
+    SKIP_EMAIL_CONFIRMATION_STEP: false,
+
+    /**
+     * Disables certain features of the application. Set a property to false to
+     * restrict usage of the feature by all users of the system
+     */
+    FEATURE_FLAGS: {
```

Comment:
Feature flags could also be updated after you update the branch to the latest master

Reply:
Yes, it was renamed as 'featureFlags' in config but is used as 'FEATURE_FLAGS' in router and sidebar on master